scripts/scraper.R

```diff
@@ -0,0 +1,123 @@
+library(pkgsearch)
+library(gh)
+pak::pak("Genentech/rd2markdown")
+## Helper function to ensure dirs exist
+write_content_to_file <- function(content, file_path, is_binary = FALSE) {
+  # Write the content to the file while checking for binaryness
+  if (!is_binary) {
+    dir_path <- dirname(file_path)
+    # If dir doesn't exist, create it
+    if (!dir.exists(dir_path)) {
+      dir.create(dir_path, recursive = TRUE)
+    }
+    writeLines(rawToChar(content), file_path)
+  }
+}
+
+# Helper function to determine if file extension indicates binary
+is_binary_file <- function(file_path) {
+  binary_extensions <- c(
+    "png",
+    "jpg",
+    "jpeg",
+    "gif",
+    "pdf",
+    "zip",
+    "tar",
+    "gz",
+    "bz2",
+    "xz",
+    "exe",
+    "dll",
+    "bin"
+  )
+  file_ext <- tools::file_ext(file_path)
+  return(file_ext %in% binary_extensions)
+}
+
+# Define the URL and output file path
+ctv_url <- "https://raw.githubusercontent.com/cran-task-views/Epidemiology/main/Epidemiology.md"
+output_file <- "Epidemiology.md"
+
+# Download the file from the URL and save it to the specified path
+download.file(ctv_url, output_file)
+
+# Read the CTV file and extract package names
+pkgs <- ctv::read.ctv("Epidemiology.md") |>
+  purrr::pluck("packagelist", "name") |>
+  pkgsearch::cran_packages() |>
```

Comment:
AFAICT, this is not necessary since only the package name, which we already have, is used thereafter

Reply:
Thanks! Fixed in 220aebe 😊